docusaurus/docs/iOS/guides/push-notifications.md

```diff
@@ -1,3 +1,104 @@
 ---
 title: Push Notifications
----
\ No newline at end of file
+---
+
+# Push Notifications
+
+First step on a way to setting up push notifications is authentication.
+Stream supports both **Certificate-based provider connection trust (.p12 certificate)** and **Token-based provider connection trust (JWT)**. Token based authentication is the preferred way to setup push notifications. This method is easy to setup and provides strong security. You can find more on setting up authentication [here](https://getstream.io/chat/docs/php/push_ios/?language=swift).
+
+## Testing Push Notifications setup
+
+Once authentication is taken care of, you can test your setup as explained below.
+
+First of all make sure you have at least one [device associated](#managing-devices-for-testing-purposes).
+
+If the device you want to test notifications on is already added to the list, you can now test pushes:
+
+    stream chat:push:test
```

Comment:
Wrapping these in code blocks?

Reply:
Makes sense 👍

Reply:
On the second thought there is no keywords of one of the known languages to recognise, so this makes no difference

Reply:
looks better stylistically though

Reply:
Do you mean in the source code or in the rendered results? Because it renders exactly the same, for instance here the upper one is just the usual markdown code block indent and the lower one is the triple ticks code block
<img width="731" alt="Screenshot 2021-06-16 at 14 32 11" src="https://user-images.githubusercontent.com/7489425/122211852-dbf81780-ceaf-11eb-862e-5afc2ee98801.png">

Reply:
OK my bad, haven't realized this was the case 👍